src/app/pages/recipes/recipes.component.ts

```diff
@@ -1,11 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+import { RecipeService } from './../../shared/services/recipe.service';
 import { Recipe } from './recipe.model';
-import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-recipes',
   templateUrl: './recipes.component.html',
   styleUrls: ['./recipes.component.scss']
 })
-export class RecipesComponent  {
+export class RecipesComponent implements OnInit {
   selectedRecipe!: Recipe;
+
+  constructor(private recipeService: RecipeService){}
+
+  ngOnInit() {
```

Comment:
If you don't want to use selected recipe inside component, you can use async pipe in html
 [recipe]="recipeService.recipeSelected | async"